Applications/CMC/test/testCMCArm26_Thelen.cpp

```diff
@@ -41,10 +41,27 @@ void testCMCArm26() {
     Storage *standard = new Storage();
     cmc.getModel().formStateStorage(temp, *standard);
 
-    Array<double> rms_tols(0.02, 2*2+2*6); // activations within 2%, angles within .6 degrees
+    Array<double> rms_tols(0, 2*2+2*6); // activations within 2%, angles within .6 degrees
     const string& muscleType = cmc.getModel().getMuscles()[0].getConcreteClassName();
     string base = "testCMCArm26 "+ muscleType;
 
+    rms_tols[0]  = 0.0002;    // r_shoulder/r_shoulder_elev/value
+    rms_tols[1]  = 0.002;     // r_shoulder/r_shoulder_elev/speed
+    rms_tols[2]  = 0.0006;    // r_elbow/r_elbow_flex/value
+    rms_tols[3]  = 0.006;     // r_elbow/r_elbow_flex/speed
+    rms_tols[4]  = 0.0000001; // TRIlong/activation
```

Comment:
> I just looked at the rms errors manually after I changed the simulation times and set these values to be quite tight.

Ok, seems fine provided all platforms arrive at the same answer—which may not be the case (IIRC, different LAPACK versions on different platforms can cause slightly different optimizer behavior). Another option could be to update the standards based on this new time range and set all tolerances to the same value.